### DISCUSSION TITLE
[Question]: Is there a way to filter Home Assistant Notifications to just Alerts with 0.14.0-beta4?

### DISCUSSION THREAD

**Discussion by rwlove** [blakeblackshear/frigate#12539] [Ask A Question] [answered]
### What is your question:

I'd prefer to only get notifications on Alerts and not Detections. I would assume this is one of the intents behind separating Alerts and Detections in 0.14.0, but maybe the code just isn't there yet. Is there a not-too-hacky-way to do this currently?

**Comment by NickM-27** [comment, marked as answer]
Just add a condition to the automation that checks that the severity is `alert`
